units/spawnhost_expiration_check.go

```diff
@@ -86,4 +90,37 @@ func (j *spawnhostExpirationCheckJob) Run(ctx context.Context) {
 		j.AddError(errors.Wrapf(err, "extending expiration for host '%s'", j.HostID))
 		return
 	}
+	// If an unexpirable host hasn't been used in a while, send an email encouraging the user to remove the host.
+	j.AddError(tryIdleSpawnHostNotification(j.host))
+
+}
+
+// tryIdleSpawnHostNotification attempts to insert a subscription and notification for this spawn host.
+func tryIdleSpawnHostNotification(h *host.Host) error {
+	shouldNotify, err := h.ShouldNotifyStoppedSpawnHostIdle()
+	if err != nil || !shouldNotify {
+		return err
+	}
+	usr, err := user.FindOneById(h.StartedBy)
+	if err != nil {
+		return errors.Wrapf(err, "finding user '%s'", h.StartedBy)
+	}
+	if usr == nil {
+		return errors.Errorf("user '%s' not found", h.StartedBy)
+	}
+	subscriber := event.NewEmailSubscriber(usr.Email())
+	subscription := event.NewSpawnHostIdleWarningSubscription(h.Id, subscriber)
+	if err = subscription.Upsert(); err != nil {
+		return errors.Wrap(err, "upserting idle spawn host subscription")
+	}
+	grip.Info(message.Fields{
+		"message":                 "sending idle host notification",
+		"host_id":                 h.Id,
+		"owner":                   h.StartedBy,
+		"last_communication_time": h.LastCommunicationTime,
```

Comment:
Last stop event time rather than last communication time.

Or maybe we don't need this log? It looks kind of like it's for debugging only.